Client/Application/AppDelegate.swift

```diff
@@ -246,7 +238,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         singleShotTimer.resume()
         shutdownWebServer = singleShotTimer
         backgroundWorkUtility?.scheduleOnAppBackground()
-        tabManager.preserveTabs()
```

Comment:
Note: removal of this was intentional, it has been moved into `BrowserViewController.swift`